src/components/PhoneInput/countriesDropdown/countriesList.js

```diff
@@ -0,0 +1,62 @@
+import React, { memo } from 'react';
+import PropTypes from 'prop-types';
+import RenderIf from '../../RenderIf';
+import Icon from '../../PicklistOption/icon';
+import StyledIconContainer from '../../PicklistOption/styled/iconContainer';
+import StyledCheckmarkIcon from '../../PicklistOption/styled/checkmarkIcon';
+import { StyledItem, StyledCountryCodeItem } from './styled';
+import { useSimulatedLoading } from './hooks';
+
+const CountriesList = memo(props => {
+    const { countries, country, itemsRef, handleCountryChange, handleActiveChange } = props;
```

Comment:
normally handle is used to define the function that handles the callback, instead this should be called onCountryChange and onActiveChange, I think it is better simply call them onChange and onHover